src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -22,6 +27,12 @@ export class SlashCommandBuilder {
 	 */
 	public readonly options: ToAPIApplicationCommandOptions[] = [];
 
+	/**
+	 * Whether the command is enabled by default when the app is added to a guild
+	 * @default true
+	 */
+	public readonly defaultPermission: boolean = undefined!;
```

Comment:
Every other property that does `undefined!` does it because that property is mandatory! This one isn't, so we can safely mark it as potentially undefined

```suggestion
	public readonly defaultPermission: boolean | undefined = undefined;
```